### PR TITLE
'loose up' some deps in composer.json

### PR DESCRIPTION
**Pull request by narcoticfresh**
the purpose is to have a recent phpunit release.. also we need to switch the rabbitmq stuff as it changed organization on github

### CODE REVIEW

**Comment by narcoticfresh** [comment]
retest this please

**Comment by narcoticfresh** [comment]
retest this please

File: composer.json [hunk 1]
```diff
@@ -59,13 +59,13 @@
         "symfony/psr-http-message-bridge": "~0.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.6",
+        "phpunit/phpunit": "*",
```
Comment (hairmare) [inline composer.json]: Can we have a `>=` constraint here instead. Otherwise composer will take way too much releases into account.

**Comment by narcoticfresh** [comment]
retest this please

File: app/config/config_test.yml [hunk 2]
```diff
@@ -48,10 +48,3 @@ old_sound_rabbit_mq:
     producers:
         document_event:
             class: Graviton\RabbitMqBundle\Producer\Dummy
-
-graviton_proxy:
-    sources:
-        swagger:
-            graviton:
-                prefix: graviton
-                uri:    http://localhost:8000/swagger.json
```
Comment (narcoticfresh) [inline app/config/config_test.yml]: @hairmare this was the culprit of stuff like `rqlIsIgnored` tests mysteriously failing - whenever `MainController` was requested in a test, this couldn't be retrieved and the test failed..

I'm confused - I'm aware that @samsamann wanted to test 'our swagger' somewhere - but removing it doesn't cause any test failures(?) so it seems to be really obsolete..

sadly, this didn't better the `coverage-clover` performance in Jenkins

Comment (hairmare) [inline app/config/config_test.yml]: I had some issues with this as well but didn't find the time to muck about with it long enough.

I'll do some profiling as well as soon as qt4/qcachegrind have compiled on my machine.

BTW: Did you see the docs in [the testbundles readme](src/Graviton/TestBundle/README.md)?

**Comment by narcoticfresh** [comment]
retest this please

**Pull request closed without merging**